task/3scale-toolbox-import/0.1/OWNERS

```diff
@@ -0,0 +1,3 @@
+# The OWNERS file is used by prow to automatically merge approved PRs.
+
+approvers:
```

Comment:
are you willing to add yourself as the owner of this Task? If so, can you please add your GitHub handle over here?

Reply:
Added my users as an owner